copy/copy.go

```diff
@@ -169,6 +185,10 @@ type Options struct {
 	// Download layer contents with "nondistributable" media types ("foreign" layers) and translate the layer media type
 	// to not indicate "nondistributable".
 	DownloadForeignLayers bool
+
+	// When only a subset of images of a list is copied, this action indicates if the manifest should be kept or stript.
```

Comment:
Not often that someone sneaks a new word past me, but you did with "stript".   It's fine, but I wonder if more people would understand "stripped" even though they're equivalent....

Reply:
Fixed. It wasn't intentional and I used 'stripped' elsewhere. But it's good to know that others may learn from my lack of knowledge of the English language.